### PR TITLE
Sign In CSS

### PR DESCRIPTION
**Pull request by cngustjr1235**
* Add material modules at app.module.ts
* Modified sign in status
* Add sign in component CSS
* Modified initial `joinedPartyId` to `0` at lobby component
* Add Snack Bar when Wrong Input

### CODE REVIEW

File: frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { UserItemComponent } from './party/party-ordered/user-item/user-item.com
 import { VoteRestaurantComponent } from './party/party-choosing-restaurant/vote-restaurant/vote-restaurant.component';
 
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { MatInputModule } from '@angular/material';
+import {MatButtonModule, MatInputModule, MatSnackBarModule} from '@angular/material';
```
Comment (pbzweihander) [inline frontend/src/app/app.module.ts]: Please add spaces after and before brackets.

File: frontend/src/app/app.module.ts [hunk 2]
```diff
@@ -51,7 +51,9 @@ import { MatInputModule } from '@angular/material';
       headerName: 'X-CSRFToken'
     }),
     BrowserAnimationsModule,
-    MatInputModule
+    MatInputModule,
+    MatButtonModule,
+    MatSnackBarModule
```
Comment (pbzweihander) [inline frontend/src/app/app.module.ts]: Please add trailing comma.

File: frontend/src/app/sign-in/sign-in.component.spec.ts [hunk 3]
```diff
@@ -7,6 +7,8 @@ import { RouterTestingModule } from '@angular/router/testing';
 
 import { SignInComponent } from './sign-in.component';
 import { UserService } from '../services/user.service';
+import {MatButtonModule, MatInputModule, MatSnackBarModule} from '@angular/material';
```
Comment (pbzweihander) [inline frontend/src/app/sign-in/sign-in.component.spec.ts]: Please add spaces after and before brackets.

File: frontend/src/app/sign-in/sign-in.component.spec.ts [hunk 4]
```diff
@@ -25,11 +27,16 @@ describe('SignInComponent', () => {
...
+
     TestBed.configureTestingModule({
       imports: [
         FormsModule,
         HttpClientTestingModule,
         RouterTestingModule,
+        MatInputModule,
+        MatButtonModule,
+        BrowserAnimationsModule,
+        MatSnackBarModule
```
Comment (pbzweihander) [inline frontend/src/app/sign-in/sign-in.component.spec.ts]: Please add trailing comma.

File: frontend/src/app/sign-in/sign-in.component.ts [hunk 5]
```diff
@@ -27,7 +27,8 @@ export class SignInComponent implements OnInit {
 
   constructor(
     private userService: UserService,
-    private router: Router
+    private router: Router,
+    public snackBar: MatSnackBar
```
Comment (pbzweihander) [inline frontend/src/app/sign-in/sign-in.component.ts]: Please add trailing comma.

**Comment by cngustjr1235** [comment]
Requests resolved

**Pull request merged**